package.json

```diff
@@ -32,9 +32,8 @@
   },
   "dependencies": {
     "@openstfoundation/mosaic-contracts": "^0.10.0-alpha.1",
-    "bignumber.js": "4.1.0",
+    "bn.js": "4.11.8",
     "js-scrypt": "^0.2.0",
-    "node-cmd": "3.0.0",
     "shelljs": "0.8.1",
```

Comment:
For what do we use `shelljs`?

Reply:
I have removed it, I think it was used for the previous web pack thing.